projects/packages/changelogger/lib/KeepAChangelogParser.php

```diff
@@ -178,7 +178,7 @@ function ( $m ) {
 			if ( '[' === $version[0] && ']' === substr( $version, -1 ) ) {
 				$version = substr( $version, 1, -1 );
 				if ( ! isset( $links[ $version ] ) ) {
-					throw new InvalidArgumentException( "Heading seems to have a linked version, but link was not found: $heading" );
+					throw new InvalidArgumentException( "Heading seems to have a linked version, but link was not found: $heading. Please ensure this version has a GitHub compare URL at the end of the CHANGELOG.md." );
```

Comment:
This is in generic parser code. It might not be a GitHub compare URL (could be some other URL), and the file might not be named CHANGELOG.md.

Reply:
So, maybe
```suggestion
					throw new InvalidArgumentException( "Heading seems to have a linked version, but link was not found: $heading. Please ensure this version has a URL defined at the end of the changelog." );
```